src/commons/UI/BaseExpandableTextArea.vue

```diff
@@ -23,18 +23,40 @@ export default defineComponent({
     isError: {
       type: Boolean,
       default: false
+    },
+    minHeight: {
+      type: String,
+      default: "30px"
+    },
+    placeholder: {
+      type: String
     }
   },
   emits: [UPDATE_MODEL_VALUE],
-  setup(_, context) {
-    const input = ref("");
+  setup(props, context) {
+    const textAreaRef = ref<null | {
+      style: { height: string };
+      scrollHeight: number;
+    }>(null);
+
+    function adjustHeight() {
+      if (textAreaRef.value) {
+        textAreaRef.value.style.height = props.minHeight;
+        textAreaRef.value.style.height = textAreaRef.value.scrollHeight + "px";
+      }
+    }
+
+    onMounted(() => {
+      adjustHeight();
+    });
 
-    function userInput(event: InputEvent) {
-      const target: HTMLElement = event.target as HTMLElement;
-      context.emit(UPDATE_MODEL_VALUE, target.innerHTML);
+    function userChange(event: Event) {

```

Comment:
Please check whether can we use watch for adjustHeight and can removed onMounted?